spec/models/product_user_creator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "spec_helper"
+
+describe ProductUserCreator do
+  subject do
+    -> { ProductUserCreator.create(user: user, product: product, approver: approver) }
```

Comment:
I don't remember where I picked it up but it's something I used a bit on another project, mainly because it made things like `it { is.expected_to change(Something, :method).from(old).to(new) }` possible. But that's not something we'll use here yet anyway because we need to upgrade `rspec`…